wallet/udb/addressmanager.go

```diff
@@ -576,7 +576,7 @@ func (m *Manager) AccountExtendedPubKey(dbtx walletdb.ReadTx, account uint32) (*
 // unlocked for this operation to complete.
 func (m *Manager) AccountExtendedPrivKey(dbtx walletdb.ReadTx, account uint32) (*hdkeychain.ExtendedKey, error) {
 	if account == ImportedAddrAccount {
-		return nil, errors.E(errors.Invalid, "imported account has no extended pubkey")
+		return nil, errors.E(errors.Invalid, "imported address account has no extended pubkey")
```

Comment:
```suggestion
		return nil, errors.E(errors.Invalid, "imported address account has no extended privkey")
```

Also, shouldn't this check be for `account >= ImportedAddrAccount`?